### PR TITLE
Cli fix update-models default module flag value

### PR DESCRIPTION
**Pull request by zaporter-work**
Surprisingly, v2 of our cli library does not auto populate empty string flags (and ignores the default value).

This causes `update-models` to try and open "" (and fail) if the `--module` flag is not provided.

### CODE REVIEW

**Comment by benjirewis** [comment]
> Surprisingly, v2 of our cli library does not auto populate empty string flags (and ignores the default value).

Weird... Does the same behavior exist with `module upload` and `module update` or just `module update-models`?

**Comment by zaporter-work** [comment]
Yeah, this code is present in those actions as well.

I've debated refactoring into a helper func but IIRC, v3 of our cli tool (coming out soon) might fix this

**Comment by zaporter-work** [comment]
I just found an issue on their github about this. Gimme a sec to edit this PR

File: cli/module_registry.go [hunk 1]
```diff
@@ -159,7 +155,7 @@ func UpdateModuleAction(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	printf(c.App.Writer, "Module successfully updated! You can view your changes online here: %s\n", response.GetUrl())
+	printf(c.App.Writer, "Module successfully updated! You can view your changes online here: %s", response.GetUrl())
```
Comment (zaporter-work) [inline cli/module_registry.go:158]: flyby

**Review by zaporter-work — COMMENTED** [review]
Swapped to use `Value` instead of `DefaultText`.

`Value` also creates the `(default:"./meta.json")` help message, but it also replaces the value if it is not present.

**Review by benjirewis — APPROVED** [review]
LGTM after offline discussion.

**Review by benjirewis — APPROVED** [review]
Nice thanks for the other flyby changes.

**Pull request merged**